### DISCUSSION TITLE
vi/start/join

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#729] [Announcements]
# vi/start/join

Giới thiệu Chào mừng đến với cộng đồng của chúng tôi! Để tham gia vào máy chủ Discord và bắt đầu sử dụng Korepi, hãy làm theo các bước sau. Bước 1: Tạo tài khoản Discord nếu bạn...

https://korepi.com/vi/start/join.html

**Comment by Pymzkhumngu** [comment]
The invite link has been expired
